### PR TITLE
Add Zero Horizontal Line on Delta B Chart

### PR DESCRIPTION
**Pull request by k-xo**
Addresses #267

### CODE REVIEW

File: projects/ui/src/components/Common/Charts/ChartPropProvider.tsx [hunk 1]
```diff
@@ -135,6 +135,7 @@ export type BaseChartProps = {
   curve?: CurveFactory | keyof typeof CURVES;
   scale?: keyof typeof SCALES;
   isTWAP?: boolean;
+  isZeroHorizontal?: boolean;
```
Comment (silochad) [inline projects/ui/src/components/Common/Charts/ChartPropProvider.tsx:138]: @k-xo Can you walk me thru why you chose this route instead condensing both into one var like `horizontalLine: number`?

File: projects/ui/src/components/Common/Charts/MultiLineChart.tsx [hunk 2]
```diff
@@ -266,7 +267,24 @@ const MultiLineChart: React.FC<BaseChartProps> = (props) => (
             {...providerProps}
             {...props}
           >
-            {(childProps) => <ExploitLine {...childProps} />}
+            {(childProps) => (
+              <>
+                <ExploitLine {...childProps} />
+                {props.isZeroHorizontal && (
+                  <Line
+                    from={{ x: 0, y: childProps.scales[0].yScale(1) as number }}
```
Comment (silochad) [inline projects/ui/src/components/Common/Charts/MultiLineChart.tsx:275]: Believe this will put a value at `y = 1` instead of `y = 0`

**Review by silochad — CHANGES_REQUESTED** [review]

File: projects/ui/src/components/Common/Charts/ChartPropProvider.tsx (hunk 1 above)
Comment (k-xo) [reply projects/ui/src/components/Common/Charts/ChartPropProvider.tsx:138]: Ah yes, was getting the zeroHorizontal approach to work, but as mentioned final version would be better with a generalised version, will make this change now

**Review by silochad — APPROVED** [review]

**Comment by silochad** [comment]
Thank you @k-xo! 🚀

**Pull request merged**